pages/1.13/gui/secrets/index.md

```diff
@@ -10,6 +10,66 @@ model: /1.13/data.yml
 ---
 
 You can manage secrets and certificates from the Secrets page.
-![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 
+![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 Figure 1 - Secrets page
+
+
+For complete details on creating and managing Secrets, see the [Secrets](/1.13/security/ent/secrets) documentation.
+
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
+
+1. Open the **Secrets** tab.
+
+1. Click the **+** icon in the top right.
+
+    ![New Secret](/1.13/img/new-secret.png)
+
+    Figure 2. New Secret icon
+
+1. In the **ID** box, provide the name of your secret and its path, if any.
+
+1. Type or paste the secret into the **Value** box.
+
+    ![Secret ID/Value Fields](/1.13/img/create-secret.png)
+
+    Figure 3. Creating a new Secret
+
+1. Click **Create**.
+
+# Creating secrets from a file 
+
```

Comment:
Should we include the maximum size warning here too? We have it in the `pages/1.13/security/ent/secrets/create-secrets/index.md` file.

Reply:
Just to be clear, are you talking about this warning?

`IMPORTANT: The maximum file size for a secret is approximately one MB, subtracting approximately one KB for the secret store metadata.`

Reply:
That’s the one!